### DISCUSSION TITLE
Allow issue authers to change labels

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#4755] [Ideas] [answered]
I'm not sure if this is even possible, but wouldn't it be useful if pr and issue authors could change their labels?

**Comment by Weredime** [comment, marked as answer]
I'm pretty sure GitHub doesn't support this yet. 

A workaround is for @WorldLanguages to actually invite people to the org, but we all know that's never going to happen soon

**Reply by Samq64** [reply]
I'm guessing it's the same with self-assigning, milestones, and projects?

**Reply by Weredime** [reply]
> I'm guessing it's the same with self-assigning, milestones, and projects?

Yup

**Reply by mxmou** [reply]
I don't think GitHub needs to support it: labels are meant for project maintainers to organize issues and PRs. Contributors can help with that by choosing the correct issue template but it wouldn't make sense to let them change things like priority or milestones.

**Reply by BroJac5246** [reply]
Could we have a feature to let the creators somehow tell a bot to add select tags?

**Comment by Hans5958** [comment]
I know that this is possible with a bot, like a command such as `/add this cool label`, but I'm not sure if there are GitHub Actions to do that. But before that, you need to tell the rationale and what should be able to be edited. Obviously, it can't be *all of the labels*.

> I don't think GitHub needs to support it: labels are meant for project maintainers to organize issues and PRs.

You could also argue that on Discord roles. Yet, there are bots who allow people to get specific roles as they wish (read "self-assignable roles" or "self roles").
